addons/udes_sale_stock/tests/common.py

```diff
@@ -23,13 +22,15 @@ def create_sale_line(cls, sale, product, qty, **kwargs):
         return cls.env["sale.order.line"].create(create_values)
 
     @classmethod
-    def create_sale(cls, customer, **kwargs):
+    def create_sale(cls, customer, client_order_ref, requested_date, **kwargs):
```

Comment:
Question: Is this method called always with client_order_ref and requested date on other classes which inherits from BaseSaleUDES?